src/commands/external.rs

```diff
@@ -66,7 +67,15 @@ pub async fn execute_external_subcommand(
     )
     .await?;
 
-    let mut command = Command::new(plugin_store.installed_binary_path(&plugin_name));
+    let binary = plugin_store.installed_binary_path(&plugin_name);
+    if !Path::new(binary.as_os_str()).exists() {
+        return Err(anyhow!(
+            "plugin executable {:?} is missing. Try uninstalling and installing the plugin again.",
```

Comment:
Use the `common::ui::quoted_path()` function (`anyhow!("plugin executable {} ...", quoted_path(&binary))`) - this ensures consistent formatting and correct rendering across platforms.  (We found that the debug format prints double backslashes on Windows.)

Reply:
thank you ❤️